src/rise-twitter.js

```diff
@@ -19,12 +19,15 @@ export default class RiseTwitter extends HTMLElement {
     this.isPreview = false;
```

Comment:
Most of these changes are from previous PRs. Not sure why they're being registered as differences.